index.html

```diff
@@ -1497,6 +1497,92 @@ <h2>Refreshing</h2>
 
       </section>
 
+      <section>
+       <h2> Mode of Operation</h2>
+
+<em>This section is non-normative.</em>
+
+<p>
+Section <a href="#Ecosystem Overview"></a> provided an overview of the verifiable 
+credential eco-system. A more detailed look at the way the eco-system is envisaged 
+to operate is as follows.
+</p>
+
+      <p>
+Note that the following ordering of steps is not fixed and each step may be repeated 
+multiple times.
+      </p>
+
+      <ul style="list-style-type:none">
+        <li>
+Step 1. The verifier defines its policies for accepting verifiable credentials 
+from holders for its supported actions.
+   </li>
+   <li>
+Step 2. The issuer issues a verifiable credential to the holder.
+  </li>
+   <li>
+Step 3. The holder MAY pass on its verifiable credentials to another holder.
+  </li>
+   <li>
+Step 4. The final holder presents its verifiable credentials to the verifier in 
+a verifiable presentation, requesting a supported action.
+  </li>
+   <li>
+Step 5. The verifier verifies the authenticity of the verifiable presentation
+and verifiable credentials.
+  </li>
+   <li>
+Step 6. The verifier verifies the right of the holder to posses the verifiable 
```

Comment:
"right" seems to be a bit strong here, but don't know what to change it to.

Reply:
changed it to
The verifier verifies that the holder may posses the verifiable 
credentials.

Reply:
Perhaps it should say "Is authorized to possess"? "May possess" doesn't seem clear. Perhaps even "is authorized to present" would be better. Even that doesn't seem quite right -- it's not up to the verifier to make that determination. The verifier simply controls how it reacts to the use of the credential. So really, maybe this should be something like "The verifier verifies that the presentation of the credentials by the holder is acceptable for its intended purpose."

Reply:
>  "The verifier verifies that the presentation of the credentials by the holder is acceptable for its intended purpose."

Yes, this.

Reply:
"its intended purpose" is ambiguous. Is this the purpose of the verifier or the purpose of the issuer, or the purpose of the subject etc. Furthermore, the next step (7) checks what you are proposing. All step 6 is checking is if step 3 took place or not, and if it did, if it conforms to the standard.
How about "The verifier verifies that the holder may posses the verifiable credentials in conformance to the standard."
Rationale: We are testing whether the verifier conforms to the standard or not, not whether the verifier wishes to accept a VC or not. A verifier can accept all VCs and all VPs, even ones that have expired and that have been revoked.